app/views/shared/_email_subscribe_unsubscribe_flash.html.erb

```diff
@@ -1,29 +1,25 @@
-<% if @account_flash.include?("email-subscription-success") %>
+<% banner_heading = capture do %>
```

Comment:
I think this should go into a helper function as well to keep the view smaller - either in the helper class [suggested here](https://github.com/alphagov/government-frontend/pull/2383/files#r822405822), or as a controller method if you decide not to go with that